python_tutorial/README.md

```diff
@@ -0,0 +1,1017 @@
+    - [Writing Scripts](#writing-scripts)
+    - [Exercise: List Comprehensions](#exercise-list-comprehensions)
+    - [Beware of Indendation!](#beware-of-indendation)
+    - [Tabs vs Spaces](#tabs-vs-spaces)
+    - [Writing Functions](#writing-functions)
+    - [Advanced Exercise](#advanced-exercise)
+    - [Object Basics](#object-basics)
+      	- [Defining Classes](#defining-classes)
+      	- [Using Objects](#using-objects)
+    - [Static vs Instance Variables](#static-vs-instance-variables)
+    - [More Python Tips and Tricks](#more-python-tips-and-tricks)
+    - [Troubleshooting](#troubleshooting)
+    - [More References](#more-references)
+- [Exercises](#exercises)
+    - [Question 1: Addition](#question-1-addition)
+    - [Question 2: buyLotsOfFruit function](#question-2-buylotsoffruit-function)
+    - [Question 3: shopSmart function](#question-3-shopsmart-function)
+
+## Tools
+In order to work with Python, we encourage you to use a text editor and a terminal. Make sure to read the whole tools section before starting your installation.
+
+On Unix and OS X, you can use the basic terminal while on windows, you will use the *Anaconda prompt* (see further).
+
+There are plenty of text edtior that can be used but [Atom](https://atom.io/) or [Sublime Text](https://www.sublimetext.com/) are good candidates.
+
+Also, there exist many IDEs such as [Spyder](https://www.spyder-ide.org/) or [PyCharm](https://www.jetbrains.com/pycharm/) that could be used **CAREFULLY**.
+
+### Python
+We recommand you to install python 
+[Conda](https://docs.anaconda.com/anaconda/install/). It allows you to manage many environments, with different Python versions and dependencies.
+
+For windows users, Conda comes with the *Anaconda prompt* that will be used as a terminal to execute Python code.
+
+### Unix & OS X basics
+When you open a terminal window, you’re placed at a command prompt:
+
+```
+MacBook-Pro-de-Pascal:~ pascalleroy$ 
+```
+
+The prompt shows your username, the host you are logged onto, and your current location in the directory structure (your path). The tilde character is shorthand for your home directory. Note your prompt may look slightly different. To make a directory, use the `mkdir` command. Use `cd` to change to that directory:
+
+```
+MacBook-Pro-de-Pascal:~ pascalleroy$ mkdir foo
+MacBook-Pro-de-Pascal:~ pascalleroy$ cd foo
+MacBook-Pro-de-Pascal:foo pascalleroy$ 
+```
+
+Use `ls` to see a listing of the contents of a directory, and `touch` to create an empty file:
+
+```
+MacBook-Pro-de-Pascal:foo pascalleroy$ ls
+MacBook-Pro-de-Pascal:foo pascalleroy$ touch hello
+MacBook-Pro-de-Pascal:foo pascalleroy$ ls
+hello
+MacBook-Pro-de-Pascal:foo pascalleroy$ cd ..
+MacBook-Pro-de-Pascal:~ pascalleroy$ 
+```
+
+Download **[LINK TO ZIP]()** into a directory and change your terminal to that directory(note: the zip file’s name may be slightly different when you download it). Use `unzip` to extract the contents of the zip file:

```

Comment:
change this

Reply:
"Download **[LINK TO ZIP]()** into a directory of your choice. On your terminal, change the current directory to that chosen directory. Use `unzip` to extract the contents of the zip file:"